packages/map/package.json

```diff
@@ -11,7 +11,7 @@
   "bugs": {
     "url": "https://github.com/goodhood-eu/goodhood/issues"
   },
-  "version": "7.0.1",
+  "version": "7.0.2-beta.0",
```

Comment:
Personal opinion: I would never commit beta versions, only the final version. Even when publishing intermediate beta versions to npm.
And with [yalc](https://wiki.nebenan.de/display/AWSM/How+to+work+with+1st+party+packages) this might not even be necessary.

Reply:
So how could I get this PR into the Core through QA without changing anythign at all after QA and without releasing an untested version of this package? Even if I dont change the version here and use the branch-name in the dependencies of the nebenan-core, I would have to commit/change something in Core after QA tested the core-branch (since there would be the master branch name used and not the final version name)